Cargo.toml

```diff
@@ -65,3 +65,37 @@ required-features = ["webview"]
 [[example]]
 name = "ios-beta"
 required-features = ["uikit", "autolayout"]
+
+[[example]]
+name = "calculator"
+required-features = ["appkit"]
+[[example]]
+name = "todos_list"
+required-features = ["appkit"]
+[[example]]
+name = "animation"
+required-features = ["appkit"]
+[[example]]
+name = "autolayout"
+required-features = ["appkit"]
+[[example]]
+name = "custom_image_drawing"
+required-features = ["appkit"]
+[[example]]
+name = "text_input"
+required-features = ["appkit"]
+[[example]]
+name = "defaults"
+required-features = ["appkit"]
+[[example]]
+name = "frame_layout"
```

Comment:
I'm back and forth on whether this one should disable `autolayout` - it's feature flagged for platforms that might not have that, but can still do frame-based layout. I _think_ if `autolayout` is disabled it may also break the example at the moment, so I can PR this one after this is merged.